docs/src/components/App/Header/Header.less

```diff
@@ -1,15 +1,7 @@
 @import (reference) "~seek-style-guide/theme";
 
-.headerPlaceholder {
-  height: @grid-row-height * 8;
-}
-
 .headerBlock {
   background: @sk-blue;
-  position: fixed;
-  top: 0;
-  left: 0;
-  right: 0;
   z-index: 10;
```

Comment:
With the removal of `fixed` the `z-index` could probably be removed too. Thoughts?